src/components/community/comments/components/comment-card/index.tsx

```diff
@@ -0,0 +1,168 @@
+import React, { FC, memo, useState } from 'react';
+import parse from 'html-react-parser';
+import { compose } from 'redux';
+import { Variant } from '@fellesdatakatalog/button';
+
+import translations from '../../../../../lib/localization';
+import { CommunityPost, CommunityUser } from '../../../../../types';
+
+import TimeAgo from '../../../time-ago';
+import User from '../../../user';
+import Composer from '../composer';
+import Buttons from '../buttons/styled';
+
+import SC from './styled';
+
+import {
+  useUpdateCommentMutation,
+  useDeleteCommentMutation,
+  usePostCommentMutation
+} from '../../../../../api/user-feedback-api/comments';
+
+enum Mode {
+  UPDATE,
+  REPLY,
+  NONE
+}
+interface Props {
+  post: CommunityPost;
+  currentUser?: Partial<CommunityUser>;
+  entityId: string;
+  isReply: boolean;
+  replies?: CommunityPost[];
+}
+
+const CommentCard: FC<Props> = ({
+  post,
+  currentUser,
+  entityId,
+  isReply,
+  replies
+}) => {
+  const [postComment] = usePostCommentMutation();
+  const [updateComment] = useUpdateCommentMutation();
+  const [deleteComment] = useDeleteCommentMutation();
+  const [currentMode, setCurrentMode] = useState<Mode>(Mode.NONE);
+  const [showAllReplies, setShowAllReplies] = useState(false);
+
+  const hasReplies = (replies?.length ?? 0) > 0;
+  const maxVisibleReplies = 3;
+  const croppedReplies = showAllReplies
+    ? replies?.reverse()
+    : replies?.slice(0, maxVisibleReplies)?.reverse();
+
+  const sendPost = (content: string, mode: Mode) => {
+    switch (mode) {
+      case Mode.UPDATE:
+        updateComment({ id: entityId, post: { ...post, content } });
+        break;
+
+      case Mode.REPLY:
+      default:
+        postComment({ id: entityId, post: { content, toPid: post.pid } });
+        break;
+    }
+  };
+  return (
+    <SC.CommentCard $isReply={isReply}>
+      <SC.CommentInfo>
+        <User user={post.user} />
+        <TimeAgo startTime={post.timestamp} cutOff={24 * 60 * 60 * 1000} />
```

Comment:
Skal cutOff være 24 timer? Og sikkert bra å flytte det opp til en const. ie:
```suggestion
        <TimeAgo startTime={post.timestamp} cutOff={oneDay} />
```